### ISSUE TITLE
[Idea] Being able to find clusters/contexts that have a matching k8s resource

### ISSUE THREAD

**Issue by sunny-b** [ahmetb/kubectl-foreach#15]
So I forked this and added a feature that my team has been wanting, which is a command that allows you to find a context that contains a K8s resource we're looking for. This often comes up during oncall issues where a service or deployment is acting strange but we don't know where it's located.

Ex. `kubectl foreach -- find svc foo -n bar`

This would return any and all contexts that contain this resource. Not sure if this is something you'd be interested in for upstream or find useful, but figured I'll bring it up.

**Comment by ahmetb** [comment]
I don't understand how it works. Aren’t you shelling out to a "find" plugin?

**Comment by sunny-b** [comment]
No, I changed the code so that if it detected the `find` command after the `--`, it converts it to a `get`. So `find svc foo -n bar` would turn into `get svc foo -n bar`, but it would also go down a different `find` workflow where it would only output the kube contexts where the `get` query returned something.

I.E.
```
$ kubectl foreach -- find svc foo -n bar
stage-cluster
production-cluster
```

https://github.com/ahmetb/kubectl-foreach/blob/87284f21ad2553b4ddcf49f1804fdc925ce47c54/main.go#L223-L226

https://github.com/ahmetb/kubectl-foreach/blob/87284f21ad2553b4ddcf49f1804fdc925ce47c54/main.go#L260

**Comment by ahmetb** [comment]
Interesting, so it's like `| awk '{print $1}'|sort |uniq` in this case? What would the use case for this be other than passing it to some other tool?

I didn't design `foreach` to have parseable output (though, it is possible) and I think `find` would be a case that's specific to `get`? If there's a more generalizable pattern here, I'd be happy to give it a shot.

But right now I think just seeing `foreach -- get` output gives me a good sense of which clusters is something is running on, along with their STATUS, AGE etc (from kubectl default print columns).

**Comment by ahmetb** [comment]
I think this can just be done with `kubectl foreach -- get --ignore-not-found`
